### PR TITLE
WasatchShell updates

### PR DESCRIPTION
**Pull request by mzieg**
- fixed offset gettors not returning SpectrometerResponse
- improved WasatchShell handling of SpectrometerResponse

### CODE REVIEW

File: wasatch/FeatureIdentificationDevice.py [hunk 1]
```diff
@@ -2517,7 +2518,7 @@ def get_detector_offset_odd(self): # -> SpectrometerResponse
         if value.error_msg != '':
             return value
         self.settings.eeprom.detector_offset_odd = value.data
-        return value
+        return SpectrometerResponse(data=value)
```
Comment (mzieg) [inline wasatch/FeatureIdentificationDevice.py:2521]: The two offset gettors were inconsistent with other gettors.

File: WasatchShell/wasatch-shell.py [hunk 2]
```diff
@@ -535,6 +537,13 @@ def open(self):
     def run_gettor(self, command):
         func_name = self.gettors[command]
         value = getattr(self.device.hardware, func_name)()
+
+        if isinstance(value, SpectrometerResponse):
+            if value.error_msg:
+                self.display(f"ERROR: {value.error_msg}")
+                return
+            value = value.data
```
[the remark this answers is not in the document]
Comment (samiebee43) [reply WasatchShell/wasatch-shell.py:545]: Line 545 should be `self.display(value.data)`

[the remark this answers is not in the document]
Comment (samiebee43) [reply WasatchShell/wasatch-shell.py:545]: And Line 547 should be made into an `elif`.

**Review by samiebee43 — APPROVED** [review]
Approved. I suggested two readability improvements.

File: WasatchShell/wasatch-shell.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (mzieg) [reply WasatchShell/wasatch-shell.py:545]: > And Line 547 should be made into an `elif`.

Related to above.

**Pull request merged**